packages/e2e-tests/web-api.spec.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { ScanReport, ScanRunErrorResponse, ScanRunResultResponse } from 'service-library';
+import { A11yServiceClient, A11yServiceCredential } from 'web-api-client';
+
+describe('Web Api E2E Tests', () => {
+    const timeout = 900000; // 15 minutes
+    let scanUrl: string;
+    let baseUrl: string;
+    let apiVersion: string;
+
+    let clientId: string;
+    let clientSecret: string;
+    let authorityUrl: string;
+
+    let webApiCredential: A11yServiceCredential;
+    let webApiClient: A11yServiceClient;
+
+    beforeEach(() => {
+        scanUrl = process.env.scanUrl;
+        baseUrl = process.env.baseUrl;
+        apiVersion = process.env.apiVersion;
+
+        clientId = process.env.clientId;
+        clientSecret = process.env.clientSecret;
+        authorityUrl = process.env.authorityUrl;
+
+        webApiCredential = new A11yServiceCredential(clientId, clientSecret, clientId, authorityUrl);
+        webApiClient = new A11yServiceClient(webApiCredential, baseUrl, apiVersion);
+    });
+
+    it('Validate inputs', () => {
```

Comment:
this could be in beforeAll method call. You could also check if clientId, clientSecret & authorityUrl are set. but, make sure the values are not printed if there is any error.